docsrc/getting_started/getting_started_with_windows.rst

```diff
@@ -7,133 +7,28 @@ Torch-TensorRT has community support for Windows platform using CMake
-          "installRoot": "${projectDir}\\out\\install\\${name}",
-          "cmakeCommandArgs": "-S . -B out",
-          "buildCommandArgs": "cmake --build out",
-          "ctestCommandArgs": "",
-          "variables": [
-            {
-              "name": "CMAKE_MODULE_PATH",
-              "value": "$PWD\cmake\Modules",
-              "type": "FILEPATH"
-            },
-            {
-              "name": "Torch_DIR",
-              "value": "<Path to libtorch>\share\cmake\Torch",
-              "type": "FILEPATH"
-            },
-            {
-              "name": "TensorRT_ROOT",
-              "value": "<Path to TensorRT directory>",
-              "type": "FILEPATH"
-            },
-            {
-              "name": "CMAKE_BUILD_TYPE",
-              "value": "Release",
-              "type": " STRING"
-            }
-          ]
-        }
-      ]
-    }
-
-
-Compilation
------------
-
-* Click Build -> Build All or directly press Ctrl + Shift + B
-
-Note: After successful compilation, the build artifacts will be present at buildRoot path configured.
-
-Installation
-------------
-
-* Build -> Install Torch-TensorRT
-
-Note: After successful installation, the artifacts will be present at installRoot.
-
-
-Building With Visual Studio Code
-==================================
-
-1. Install Visual Studio Code
-2. Install Build Tools for Visual Studio 2022
-
-    - Select "Desktop Development with C++"
-      > Currently, this installs MSVC v143 - 2022. There are also options to install previous 2019/2017/2015 editions of MSVC
-      > License term "1b Build Tools additional use right" allows using Build Tools to compile Open Source Dependencies
-      > Also allows using Build Tools to develop and test Open Source Dependencies, to the minor extend of ensuring compatibility with Build Tools
-
-3. Install CUDA (e.g. 11.7.1)
-
-4. Install `TensorRT` (e.g 8.5.1.7)
-
-    - Set ``TensorRT_ROOT``
-    - Add ``TensorRT_ROOT\lib`` to ``PATH``
-
-5. Install "libtorch-win-shared-with-deps-latest.zip"
-
-    - Select build targeting the appropriate CUDA version
-    - Set ``Torch_DIR``
-    - Add ``Torch_DIR\lib`` to ``PATH``
-
-6. Clone TensorRT repo
-7. Install C++ and CMake Tools extensions from MS
-
-    - Change build to ``RelWithDebInfo``
-
-8. Update ``.vscode\settings.json``
-
-    - Clean, configure, build
-
-e.g. /.vscode/settings.json
+* Open the app "x64 Native Tools Command Prompt for VS 2022" - note that Admin priveleges may be necessary
+* Ensure Bazelisk (Bazel launcher) is installed on your machine and available from the command line. Package installers such as Chocolatey can be used to install Bazelisk
+* Install latest version of Torch (i.e. with `pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu124`)
+* Clone the Torch-TensorRT repository and navigate to its root directory
+* Run `pip install ninja`
+* Run `pip install -r requirements-dev.txt`
```

Comment:
The packages in `requirements-dev.txt` seem to be more suitable for developers who write or debug codes. For end users who simply want to build the wheel, the essential packages to install are just `numpy, PyYAML, setuptools, torch, wheel`.